tests/routes/metadata_test.py

```diff
@@ -32,6 +32,13 @@ def test_put_metadata(self):
             .put(data=meta_utils.send_meta(), headers=self.auth)
         self.assert_metadata_matches("/test/artifact/dir/dir2/dir3/nest-test/_meta")
 
+    def test_put_metadata_invalid_json(self):
+        self.route_tester \
+            .metadata() \
+            .route_params(bucket_name="test", path="dir/dir2/dir3/nest-test") \
+            .expect(400) \
```

Comment:
Can we assert the body of the response? Just in case this 400 is coming from elsewhere and we don't know because the tests are passing. Or at least because we do elsewhere lol